### DISCUSSION TITLE
单行文字和多行文字挨在一起该怎么检测识别

### DISCUSSION THREAD

**Discussion by TOMORINAONAO** [PaddlePaddle/PaddleOCR#12243] [Q&A] [answered]
![1](https://github.com/PaddlePaddle/PaddleOCR/assets/51530060/885f5ebd-df70-458c-84ac-1e5b52211b09)
公文文件内容识别，有的公文的这个大标题是这样的单行之间插了多行小字，我分开标注单行和多行的字然后用了大概五六十张图片微调检测模型，但是得到的结果还是这样全部框起来了，后续识别模型就没办法准确检测了。这种情况该如何解决

**Comment by thsno02** [comment, marked as answer]
+ 识别模型训练的时候应该是 line by line 的，所以多行文本没法正确识别。
+ 这里的问题应该是检测模型，如果你可以让检测模型拆开进行，这里的微调数据可能不太够，你使用 2k 数据试试。
